### PR TITLE
First commit - story 3 task 1 complete

### PR DESCRIPTION
**Pull request by maxmclarentaylor**
The html was just an example so we could see if it works - it will be replaced by the work on story 2

### CODE REVIEW

File: startGame.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+function countdown(number) {
+    return number - 1
+}
+
+function Timer() {
+    time = 4
+    var timer = document.getElementById("timer")
+    createTimer()
+    function createTimer() {
```
Comment (olivergate) [inline startGame.js]: you have pt a function inside a function. Bring outside.

File: startGame.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+document.getElementById("button").addEventListener('click', function(){
+    setTimeout(startGame, 3000)
+    Timer()
+})
+
+ function startGame() {
```
Comment (olivergate) [inline startGame.js]: This function doesn't do anything. Get rid of it or use it.

File: startGame.css [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#timer{
```
Comment (olivergate) [inline startGame.css]: id's should not hold style elements only classes

File: startGame.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+function Timer() {
+    time = 4
+    var timer = document.getElementById("timer")
+    createTimer()
+    function createTimer() {
+        var timing = setInterval(function () {
+            if(time === 1){
+                clearInterval(timing)
+                timer.innerHTML = 'Go!!'
```
Comment (olivergate) [inline startGame.js]: why are you using innerHTML if there is no HTML

File: startGame.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
+document.getElementById("button").addEventListener('click', function(){
+    setTimeout(startGame, 3000)
```
Comment (olivergate) [inline startGame.js]: No idea what this is, please get rid of it, it is not MVP got this task.

File: startGame.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    function createTimer() {
+        var timing = setInterval(function () {
+            if(time === 1){
+                clearInterval(timing)
+                timer.innerHTML = 'Go!!'
+            } else {
+                time = countdown(time)
+                timer.textContent = time
+            }
+
```
Comment (olivergate) [inline startGame.js]: Line break unecessary

File: startGame.js [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+    time = 4
+    var timer = document.getElementById("timer")
+    createTimer()
+    function createTimer() {
+        var timing = setInterval(function () {
+            if(time === 1){
+                clearInterval(timing)
+                timer.innerHTML = 'Go!!'
+            } else {
+                time = countdown(time)
```
Comment (olivergate) [inline startGame.js]: Camel casing please

File: startGame.js [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+ function startGame() {
+ }
+
+function countdown(number) {
+    return number - 1
+}
+
+function Timer() {
+    time = 4
+    var timer = document.getElementById("timer")
```
Comment (olivergate) [inline startGame.js]: This var should not be in the function but be available globally

File: startGame.js [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+
+ function startGame() {
+ }
+
+function countdown(number) {
+    return number - 1
+}
+
+function Timer() {
+    time = 4
```
Comment (olivergate) [inline startGame.js:49]: undeclared variable

File: startGame.js [hunk 10]
```diff
@@ -0,0 +1,29 @@
+document.getElementById("button").addEventListener('click', function(){
```
Comment (olivergate) [inline startGame.js]: Preferably use jQuery selectors please

**Review by olivergate — CHANGES_REQUESTED** [review]

File: startGame.js [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+})
+
+ function startGame() {
+ }
+
+function countdown(number) {
+    return number - 1
+}
+
+function Timer() {
```
Comment (olivergate) [inline startGame.js]: Where is the documentation for this function?

File: startGame.js (hunk 8 above)
Comment (olivergate) [reply startGame.js]: Also please use Jquery, it is a preference, however it seems like everyone is using.

File: startGame.js (hunk 1 above)
Comment (olivergate) [reply startGame.js]: And then document the function

Comment (maxmclarentaylor) [reply startGame.js]: done

File: startGame.css (hunk 3 above)
Comment (maxmclarentaylor) [reply startGame.css]: done

File: startGame.js (hunk 4 above)
Comment (maxmclarentaylor) [reply startGame.js]: done

File: startGame.js (hunk 10 above)
Comment (maxmclarentaylor) [reply startGame.js]: no more jquery in this file so will leave it

File: index.html [hunk 12]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```
Comment (olivergate) [inline index.html]: PLease change the name of this index.hml to something more specific or else the merge will delete everything else under index.html

File: startGame.js [hunk 13]
```diff
@@ -0,0 +1,40 @@
...
+        } else {
+            time = countDown(time)
+            startTimer.textContent = time
+        }
+    },1000)
+}
+
+function timer() {
+    time = 4
+    var startTimer
```
Comment (olivergate) [inline startGame.js]: unused variable. What is this for? You are calling a named function within anohter function. Not sure what this function realy does. Still n o documentation?

File: startGame.js [hunk 14]
```diff
@@ -0,0 +1,40 @@
...
+})
+
+function hideStartBtn() {
+        document.getElementById("button").style.display = "none"
+    }
+function displayGameScreen() {
+    document.getElementById("gameScreen").style.display = "block"
+}
+
+
```
Comment (olivergate) [inline startGame.js:41]: Unecessary line break

File: startGame.js [hunk 15]
```diff
@@ -0,0 +1,40 @@
...
+function countDown(number) {
+    return number - 1
+}
+
+var startTimer = document.getElementById("timer")
```
Comment (olivergate) [inline startGame.js]: Declare variables at the top of the document.

**Review by olivergate — CHANGES_REQUESTED** [review]

File: startGame.js (hunk 15 above)
Comment (olivergate) [reply startGame.js]: Nice documentation.

File: startGame.js [hunk 16]
```diff
@@ -0,0 +1,68 @@
...
+})
+
+//this function hides the start button after it has been clicked
+function hideStartBtn() {
+        document.getElementById("startbutton").style.display = "none"
+}
+
+// function to make the header disappear
+function hideGameTitle() {
+    document.getElementById("title").style.display = "none"
```
Comment (gidspektor) [inline startGame.js]: this is called aboveBox now, not title

File: startGame.js [hunk 17]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+//functions makes the topsection disappear
+
+function hideTopSection() {
+    $("topSection").hide()
```
Comment (gidspektor) [inline startGame.js]: sorry this is the aboveSection now, set to display none

File: startGame.js (hunk 16 above)
Comment (gidspektor) [reply startGame.js]: hide game title isnt needed now

Comment (Skylucas) [reply startGame.js]: On line 5 you need to add psr2 changes

File: startGame.js [hunk 18]
```diff
@@ -0,0 +1,68 @@
...
+//function that allows us to countdown number in next function
+function countDown(number) {
+    return number - 1
+}
+
+//the function for the timer
+function createTimer() {
+    time = 4
+    var timing = setInterval(function () {
+        if(time === 1){
```
Comment (Skylucas) [inline startGame.js:51]: line 52 also needs psr2 changes

File: startGame.js [hunk 19]
```diff
@@ -0,0 +1,68 @@
...
+
+//functions makes the topsection disappear
+
+function hideTopSection() {
+    $("topSection").hide()
+}
+
+    //this means the gameScreen is shown when clicked and has a fade in option
+function displayplayScreen() {
+    var $playScreen = $("playScreen")
```
Comment (gidspektor) [inline startGame.js]: $('#playscreen')

**Review by Skylucas — CHANGES_REQUESTED** [review]
So had a look at your code, needs some psr2 comments

File: startGame.js (hunk 17 above)
Comment (gidspektor) [reply startGame.js]: also $('#aboveSection')

**Pull request merged**